components/resizable/resizable.spec.ts

```diff
@@ -93,13 +93,15 @@ describe('resizable', () => {
       zone.simulateZoneExit();
       fixture.detectChanges();
       fixture.detectChanges();
-      tick(16);
-      fixture.detectChanges();
-      expect(testComponent.width).toBe(600);
-      expect(testComponent.height).toBe(200);
-    }));
+      afterNextFrameRender(() => {
```

Comment:
It seems that `tick` does not work for event stream outside of angular, thus using `setTimeout` instead